scripts/build.py

```diff
@@ -454,6 +573,27 @@ def generate_md5_hash(filepath):
     return m.hexdigest()
 
 
+def get_filepaths_by_extensions(source_dir, file_extensions):
+    """Return list of filepaths in a directory with certain extensions.
+
+    Args:
+        source_dir: str. Root directory to be walked.
+        file_extensions: tuple(str). Tuple of file extensions.
+
+    Returns:
+        filepaths: list(str). List of filepaths with specified extensions.
+    """
+    filepaths = []
+    for root, _, filenames in os.walk(source_dir):
+        for filename in filenames:
+            filepath = os.path.join(root, filename)
+            relative_filepath = os.path.relpath(filepath, source_dir)
+            if any(filename.endswith(p) for p in file_extensions):
+                print relative_filepath
+                filepaths.append(relative_filepath)
+    return filepaths
+
+
 def get_file_hashes(directory_path):
     """Returns hashes of all files in directory tree.
```

Comment:
Probably should say something about excluding files with extensions in FILE_EXTENSIONS_TO_IGNORE, here. Important to ensure that the contract of the function is strictly correct (maybe double-check the others).

Reply:
Updated docstrings for functions that utitlize filtering by file extensions.